<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseTemp</class>
 <widget class="QWidget" name="chooseTemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(32, 32, 35, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Temp180">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Temp200">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Temp220">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Temp240">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_Temp260">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_TempOff">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
